app/controllers/courses_controller.rb

```diff
@@ -48,6 +43,11 @@ def get_visible
     end
   end
 
+  def students
+    stds = User.joins("join courses_students on users.id = courses_students.student_id").where(["courses_students.course_id = ?", params[:course_id]])
```

Comment:
We cannot, because `enrolled_courses` is not a table. Indeed, hopefully Matt can provide insight here.